src/Google.Datastore.V1Beta3/DatastoreTransaction.cs

```diff
@@ -91,13 +102,13 @@ public DatastoreTransaction(DatastoreClient client, string projectId, ByteString
         /// the query is executed in the partition associated with the empty namespace in the project used by this transaction.</param>
         /// <param name="callSettings">If not null, applies overrides to RPC calls.</param>
         /// <returns>A <see cref="DatastoreQueryResults"/> representing the result of the query.</returns>
-        public DatastoreQueryResults RunQuery(Query query, PartitionId partitionId, CallSettings callSettings = null)
+        public DatastoreQueryResults RunQuery(Query query, CallSettings callSettings = null)
```

Comment:
I don't think it does - just as you can't for DatastoreDb.

In the rare case where you really want to include a query to a different partition within a transaction, you can create a new `DatastoreTransaction` for that other partition and pass in the same transaction ID. It's ugly, but I think it's nicer than making PartitionId an optional parameter here, with a different meaning to the one elsewhere (where it means "use the project ID and empty namespace").